### PR TITLE
Corrects edit_user_registration_path error by eliminating that link (#10).

### PR DESCRIPTION
**Pull request by bwatson78**
app/models/user.rb, app/views/shared/_user_util_links.html.erb, and spec/models/user_spec.rb: directly mirroring Lux code, this changes the assignment of a user's attributes and provides error protection, as well as eliminating the need for link `edit_user_registration_path` to be in the header, and updating testing expectations.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -22,12 +31,23 @@ def to_s
...
+    user.assign_attributes(display_name: auth.info.display_name)
+    # tezprox@emory.edu isn't a real email address
+    user.email = auth.info.uid + '@emory.edu' unless auth.info.uid == 'tezprox'
     user.save
     user
+  rescue User::NilShibbolethUserError => e
+    Rails.logger.error "Nil user detected: Shibboleth didn't pass a uid for #{e.auth.inspect}"
+  end
+
+  def self.log_omniauth_error(auth)
```
Comment (devanshu-m) [inline app/models/user.rb:45]: Are we using this method in this class anywhere or any other class? If not, does it make sense to remove it? I see we already have error handling above.

File: app/views/shared/_user_util_links.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<ul class="navbar-nav">
```
Comment (devanshu-m) [inline app/views/shared/_user_util_links.html.erb:1]: Is this an overwrite? If yes, can you please mention what is being overwritten?

**Review by devanshu-m — CHANGES_REQUESTED** [review]

File: app/models/user.rb (hunk 1 above)
Comment (bwatson78) [reply app/models/user.rb:45]: Yeah, good point. I'll remove it.

File: app/views/shared/_user_util_links.html.erb (hunk 2 above)
Comment (bwatson78) [reply app/views/shared/_user_util_links.html.erb:2]: And I'll add the comment in the same commit.

**Comment by bwatson78** [comment]
@devanshu-m Changes have been made and rspec is green.

**Review by devanshu-m — APPROVED** [review]

**Pull request merged**